src/views/account/UserProfile.vue

```diff
@@ -1,23 +1,76 @@
 <script setup lang="ts">
 import type { FormInst, UploadFileInfo } from "naive-ui";
-import { ref, toRaw } from "vue";
+import { ref } from "vue";
 
-import { useUserStore } from "@/stores/user";
+import { UserProfile } from "@/types/User";
+import { baseURL } from "@/utils/constants";
 import { timezones } from "@/utils/constants";
 
 const userFormRef = ref<FormInst | null>(null);
 
-const userStore = useUserStore();
+const userProfile = ref<UserProfile>({
+  username: "",
+  email_address: "",
+  first_name: "",
+  institution: "",
+  last_name: "",
+  location: "",
+  password: "",
+  profile_image: "",
+  timezone: "",
+});
+
+const rules: FormRules = {
+  username: {
+    message: "Please enter a username",
+    required: true,
+    trigger: ["blur", "input"],
+  },
+  email_address: {
+    message: "Please enter an email address",
+    required: true,
+    trigger: ["blur", "input"],
+  },
+};
+
+const loading = ref(false);
+const push = usePush();
```

Comment:
Move this to after import but before `const userFormRef = ref<FormInst | null>(null);`